docs/pages/implementation/nuts-auth.rst

```diff
@@ -0,0 +1,225 @@
+
+.. code-block::
+
+    {
+      "means": "dummy",
+      "sessionID": "490385cjalwe9587fahnly6fdu8j5r6lndr",
+      "sessionPtr": "not used"
+    }
+
+With the ``sessionID`` the current status of the signing session can be retrieved:
+
+.. code-block::
+
+    GET /internal/auth/experimental/signature/session/490385cjalwe9587fahnly6fdu8j5r6lndr HTTP/1.1
+    Host: server.example.com
+
+returns:
+
+.. code-block::
+
+    {
+      "status": "created",
+      "verifiablePresentation": "490385cjalwe9587fahnly6fdu8j5r6lndr"
+    }
+
+The ``status`` field returns means specific statuses, please consult the documentation of each means to find out what statuses are returned. The ``verifiablePresentation`` field will be filled when the session has been completed and the user has successfully signed the contract. The contents of the ``verifiablePresentation`` field is BASE64 encoded and conforms to `RFC002 <https://nuts-foundation.gitbook.io/drafts/rfc/rfc002-authentication-token>`_.
+
+Bearer token
+************
+
+The previous paragraph showed us how to get a signed contract in the form of a *verifiable presentation*. The *verifiable presentation* can be used to create a bearer token by calling:
+
+.. code-block::
+
+    POST /auth/jwtbearertoken HTTP/1.1
+    Host: server.example.com
+    Content-Type: application/json
+
+    {
+      "custodian": "urn:oid:2.16.840.1.113883.2.4.6.1:000000002",
+      "actor": "urn:oid:2.16.840.1.113883.2.4.6.1:000000001",
+      "subject": "urn:oid:2.16.840.1.113883.2.4.6.3:999999990"
+      "identity": "eyJAY29udGV4dCI6WyJodHRwczovL3d3dy53My5vcmcvMjAxOC9jcmVkZW50aWFscy92MSJdLCJwcm9vZiI6eyJCaXJ0aGRhdGUiOiIxOTgwLTAxLTAxIiwiQ29udHJhY3QiOiJFTjpQcmFjdGl0aW9uZXJMb2dpbjp2MyBJIGhlcmVieSBkZWNsYXJlIHRvIGFjdCBvbiBiZWhhbGYgb2YgT3JnIEIuIFRoaXMgZGVjbGFyYXRpb24gaXMgdmFsaWQgZnJvbSBkaW5zZGFnLCAxIGRlY2VtYmVyIDIwMjAgMTE6NDA6NTYgdW50aWwgZGluc2RhZywgMSBkZWNlbWJlciAyMDIwIDEyOjQwOjU2LiIsIkVtYWlsIjoidGVzdGVyQGV4YW1wbGUuY29tIiwiSW5pdGlhbHMiOiJJIiwiTGFzdG5hbWUiOiJUZXN0ZXIiLCJUeXBlIjoiTm9TaWduYXR1cmUifSwidHlwZSI6WyJWZXJpZmlhYmxlUHJlc2VudGF0aW9uIiwiRHVtbXlWZXJpZmlhYmxlUHJlc2VudGF0aW9uIl19Cg==",
+      "scope": "nuts"
+    }
+
+The ``scope`` field must be **nuts** according to spec and the ``identity`` field must be filled with the contents of the ``verifiablePresentation`` of the previous paragraph. The ``actor`` field must be filled with the *identifier* of the care organization the current user is representing. It must be the same as used in `Constructing a formatted contract`_. The *identifier* for the ``custodian`` field must have been obtained by other means. It can, for example, come from a registered consent record which granted this actor access to data at the custodian given a certain subject. The ``subject`` field is optional, when used it must identify a patient. ``urn:oid:2.16.840.1.113883.2.4.6.3:999999990`` is the Dutch citizen number ``999999990``.
```

Comment:
> when used it must identify a patient. ``urn:oid:2.16.840.1.113883.2.4.6.3:999999990`` is the Dutch citizen number ``999999990``.

when used it must identify a patient, e.g. ``urn:oid:2.16.840.1.113883.2.4.6.3:999999990`` is the Dutch citizen number ``999999990``.